okhttp-logging-interceptor/pom.xml

```diff
@@ -42,4 +42,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>${project.groupId}.logginginterceptor</Automatic-Module-Name>
```

Comment:
logginginterceptor –> logging ?

Reply:
we use `logging-interceptor` in the module name but just `logging` in the package name